include/zephyr/bluetooth/gatt.h

```diff
@@ -1371,6 +1380,17 @@ bool bt_gatt_is_subscribed(struct bt_conn *conn,
  */
 uint16_t bt_gatt_get_mtu(struct bt_conn *conn);
 
+/** @brief Get UATT MTU for a connection
+ *
+ *  Get negotiated UATT connection MTU, note that this does not equal the largest
```

Comment:
This could return the mtu pre-negotiation. I think you can drop the `negotiated`. 

The second part of the sentence is also a bit unclear to me: what exactly are you referring to by `packet` and `attribute data`. Is it an on-air data packet, or L2CAP PDU?

(we were reviewing at my desk w/ alwa-nordic, maybe he has a suggestion)

Reply:
Sorry for the confusion. I based this comment on the existing `bt_gatt_get_mtu`:
https://github.com/zephyrproject-rtos/zephyr/blob/878640fa0817a5c6684758d68dcb824e24571612/include/zephyr/bluetooth/gatt.h#L1480-L1489

I've updated this API description following both yours and @alwa-nordic's suggestion.